content/en/docs/Tasks/creating-operator-manifests.md

```diff
@@ -57,6 +57,7 @@ spec:
     name: Example
     url: www.example.com
   version: 0.10.0
+  minKubeVersion: 1.16.0
```

Comment:
I wonder if we'd want to proactively bump this minKubeVersion to something more recent so we don't have to update it again down the line.

/lgtm

Reply:
I changed it to 1.20.0 but I don't think it matters too much as it is a minimal version